packages/at_client/lib/src/manager/at_client_manager.dart

```diff
@@ -74,8 +75,26 @@ class AtClientManager {
   }
 
   void _notifyListeners(SwitchAtSignEvent switchAtSignEvent) {
-    for (var listener in _changeListeners) {
+    List<AtSignChangeListener> copyOfChangeListeners = List.from(_changeListeners);
+    for (var listener in copyOfChangeListeners) {
       listener.listenToAtSignChange(switchAtSignEvent);
     }
   }
+
+  /// Clears the inactive AtSignChangeListener from the _changeListeners list.
```

Comment:
Dart doc should probably say:

Removes the given listener from the list of listeners, that are notified whenever an atSign is switched.

Reply:
Sure Jagan, Will update the dart docs